charts/DataTab.tsx

```diff
@@ -34,18 +34,20 @@ export class DataTab extends React.Component<{
 
         const rows: string[] = []
 
-        const titleRow = ["Entity", "Code", "Year"]
+        const titleRow = chart.yearIsDayVar
+            ? ["Entity", "Code", "Date"]
+            : ["Entity", "Code", "Year"]
```

Comment:
nit: avoid repetition and only change the 3rd word:

    ["Entity", "Code", chart.yearIsDayVar ? "Date" : "Year"]